internal/pkg/deploy/cloudformation/stack/worker_service_integration_test.go

```diff
@@ -89,4 +92,24 @@ func TestWorkerService_Template(t *testing.T) {
 
 		require.Equal(t, string(wantedCFNParamsBytes), actualParams)
 	})
+
+	t.Run("invalid topic subscription should be rejected", func(t *testing.T) {
+		path := filepath.Join("testdata", "workloads", workerManifestPath)
+		manifestBytes, err := ioutil.ReadFile(path)
+		require.NoError(t, err)
+		mft, err := manifest.UnmarshalWorkload(manifestBytes)
+		require.NoError(t, err)
+		envMft, err := mft.ApplyEnv(subscriptionTestEnvName)
+		require.NoError(t, err)
+		v, ok := envMft.(*manifest.WorkerService)
+		require.True(t, ok)
+		badSubSerializer, err := stack.NewWorkerService(v, subscriptionTestEnvName, appName, stack.RuntimeConfig{
+			ServiceDiscoveryEndpoint: "test.my-app.local",
```

Comment:
nit: this won't be set right for a worker service?

Reply:
This gets set as an environment variable so that worker services can properly construct service discovery URLs to *other* services in the cluster.

Reply:
oh derp yea that totally makes sense